mod/projects/set/self/project/core.haml

```diff
@@ -0,0 +1,14 @@
+%div
+  = field_nest :description
+  = nest :homepage_featured_projects, items: { view: "listing" }
+%div{class: "tabbable", id: "project-tabs"}
```

Comment:
again you don't need this %div.

Just `#project-tabs.tabbable` will do.  Please change all divs with classes